src/payment/strategies/bluesnapv2/bluesnapv2-payment-page-loader.ts

```diff
@@ -0,0 +1,59 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+
+import { NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+
+import { BlueSnapV2PaymentInitializeOptions } from './bluesnapv2-payment-options';
+
+export default class BlueSnapV2PaymentPageLoader {
+    private _blueSnapV2Options?: BlueSnapV2PaymentInitializeOptions;
+    private _formPoster = createFormPoster();
+
+    initialize(options?: BlueSnapV2PaymentInitializeOptions): void {
```

Comment:
you could simplify this class by having one single method, `loadPaymentPage(iframeName, options)` but that would require to store the options in `BlueSnapV2PaymentStrategy` when initialized is called. Just a suggestion, not a blocker.

Reply:
Yeah, it makes sense, thank you!